src/main/resources/application.properties

```diff
@@ -10,9 +10,13 @@ quarkusio.localized.cn.web-uri=https://cn.quarkus.io/
 quarkusio.localized.cn.git-uri=https://github.com/quarkusio/cn.quarkus.io.git
 quarkusio.localized.ja.web-uri=https://ja.quarkus.io/
 quarkusio.localized.ja.git-uri=https://github.com/quarkusio/ja.quarkus.io.git
-# Scheduler configuration
-# At 19:00:00 every day
+
+# Indexing configuration
+fetching.parallelism=8
+fetching.timeout=10m
```

Comment:
just checking: I don't remember the exact time, but I remember it took a while to clone each site (just to make sure it fits into 10m)

Reply:
It seems to take less than 10m in prod, but we can certainly use a higher value if you think it's necessary.

Reply:
If it is ok on prod then great 😃, Maybe it just was my connection and I do not plan to reclone it locally on each start so it's also fine 😄